InnerEye/Scripts/move_model.py

```diff
@@ -96,28 +80,48 @@ def upload_model(ws: Workspace, config: MoveModelConfig) -> Model:
 
 
 def get_workspace(config: MoveModelConfig) -> Workspace:
+    """
+    Get workspace based on command line input config
+    :param config: MoveModelConfig
+    :return: an Azure ML workspace
+    """
     return Workspace.get(name=config.workspace_name, subscription_id=config.subscription_id,
                          resource_group=config.resource_group)
 
 
 def main() -> None:
-    config = MoveModelConfig.parse_args()
-    azure_config = AzureConfig.from_yaml(yaml_file_path=fixed_paths.SETTINGS_YAML_FILE,
-                                         project_root=fixed_paths.repository_root_directory())
-    ws = azure_config.get_workspace()
-    move(config, ws)
+    parser = ArgumentParser()
+    parser.add_argument("-a", "--action", type=str,
+                        help="Action (download or upload)")
+    parser.add_argument("-w", "--workspace_name", type=str, required=True,
+                        help="Azure ML workspace name")
+    parser.add_argument("-s", "--subscription_id", type=str, required=True,
+                        help="AzureML subscription id")
+    parser.add_argument("-r", "--resource_group", type=str, required=True,
+                        help="AzureML resource group")
+    parser.add_argument("-p", "--path", type=str, required=True,
```

Comment:
that's maybe the one argument where required=False could make sense, you could default to Path(.)